lib/dataloader/ecto.ex

```
@@ -304,6 +397,20 @@ if Code.ensure_loaded?(Ecto) do
 
         {key, Map.new(Enum.zip(ids, results))}
       end
+
+      defp cardinality_mapper(:many) do
+        fn value ->
+          value
+        end
+      end
+
+      defp cardinality_mapper(:one) do
+        fn
+          [] -> nil
+          [value] -> value
+          [_ | _] -> raise "expected one but got many"
```

Should maybe raise the same error as Repo.one